pprint.opam

```diff
@@ -8,13 +8,12 @@ authors: [
 homepage: "https://github.com/fpottier/pprint"
 dev-repo: "git+ssh://git@github.com/fpottier/pprint.git"
 bug-reports: "francois.pottier@inria.fr"
-build: [make "-f" "Makefile" "all"]
-install: [make "-f" "Makefile" "install"]
-remove: [make "-f" "Makefile" "uninstall"]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
 depends: [
   "ocaml" {>= "4.02"}
-  "ocamlfind" {build}
-  "ocamlbuild" {build}
+  "dune" { "1.11" }
```

Comment:
Should this be `>= "1.11"`?

Reply:
Thanks, I fixed that and put `>= "1.3"` as I tested and it worked well with this old version of dune. This should not put unnecessary pressure on people to upgrade their packages.